poco/src/dump.cpp

```diff
@@ -0,0 +1,157 @@
+#include "dump.h"
```

Comment:
Here's the actual usage of the Defold inspection api, iterating over the scene and constructing the Lua table.